src/common/base/src/runtime/runtime.rs

```diff
@@ -88,7 +88,7 @@ pub trait TrySpawn {
     ///
     /// It allows to return an error before spawning the task.
     #[track_caller]
-    fn try_spawn<T>(&self, task: T) -> Result<JoinHandle<T::Output>>
+    fn try_spawn<T>(&self, task: T, name: Option<String>) -> Result<JoinHandle<T::Output>>
```

Comment:
> nit: How about adding a `try_spawn_with_name` so we don't need to break existing code?

Hmm... I considered adding `try_spawn_with_name`(exactly the same name :)), but decided against it in favor of having fewer methods, even though it means breaking existing code. Since this is a non-public API with minimal upgrade burden, the benefits of a cleaner interface outweigh the cost of the breaking change.